airflow/providers/databricks/plugins/databricks_workflow.py

```diff
@@ -0,0 +1,472 @@
+    """Construct a link to send a repair request for a single databricks task."""
+
+    name = "Repair a single task"
+
+    def get_link(
+        self,
+        operator,
+        dttm=None,
+        *,
+        ti_key: TaskInstanceKey | None = None,
+    ) -> str:
+        if not ti_key:
+            ti = get_task_instance(operator, dttm)
+            ti_key = ti.key
+
+        task_group = operator.task_group
+        if not task_group:
+            raise AirflowException("Task group is required for generating repair link.")
+
+        self.log.info(
+            "Creating link to repair a single task for databricks job run %s task %s",
+            task_group.group_id,
+            ti_key.task_id,
+        )
+        dag = airflow_app.dag_bag.get_dag(ti_key.dag_id)
+        task = dag.get_task(ti_key.task_id)
+
+        if ".launch" not in ti_key.task_id:
+            launch_task_id = get_launch_task_id(task_group)
+            ti_key = _get_launch_task_key(ti_key, task_id=launch_task_id)
+        metadata = get_xcom_result(ti_key, "return_value")
+
+        query_params = {
+            "dag_id": ti_key.dag_id,
+            "databricks_conn_id": metadata.conn_id,
+            "databricks_run_id": metadata.run_id,
+            "run_id": ti_key.run_id,
+            "tasks_to_repair": _get_databricks_task_id(task),
+        }
+        return url_for("RepairDatabricksTasks.repair", **query_params)
+
+
+class RepairDatabricksTasks(AirflowBaseView, LoggingMixin):
+    """Repair databricks tasks from Airflow."""
+
+    default_view = "repair"
+
+    @expose("/repair_databricks_job/<string:dag_id>/<string:run_id>/", methods=("GET",))
+    @auth.has_access_dag("POST", DagAccessEntity.RUN)
+    def repair(self, dag_id: str, run_id: str):
+        if not dag_id:
+            return_url = request.referrer or url_for("Airflow.index")
+            flash("No dag_id provided. Cannot repair tasks.")
+            return redirect(return_url)
+
+        view = conf.get("webserver", "dag_default_view")
+        return_url = self._get_return_url(dag_id, view)
+
+        tasks_to_repair = request.values.get("tasks_to_repair")
+        self.log.info("Tasks to repair: %s", tasks_to_repair)
+        if not tasks_to_repair:
+            flash("No tasks to repair. Not sending repair request.")
+            return redirect(return_url)
+
+        databricks_conn_id = request.values.get("databricks_conn_id")
+        databricks_run_id = request.values.get("databricks_run_id")
+
+        if not databricks_conn_id:
+            flash("No Databricks connection ID provided. Cannot repair tasks.")
+            return redirect(return_url)
+
+        if not databricks_run_id:
+            flash("No Databricks run ID provided. Cannot repair tasks.")
+            return redirect(return_url)
+
+        if not run_id:
+            flash("No run ID provided. Cannot repair tasks.")
+            return redirect(return_url)
```

Comment:
Oh, same here. I assume if the parameters are needed, you can add of them to the URL and can save all the validation code here.

Reply:
same reasoning as above